magellan-lint/src/main/java/com/wealthfront/magellan/AvoidUsingActivity.kt

```diff
@@ -0,0 +1,50 @@
+package com.wealthfront.magellan
+
+import com.android.tools.lint.client.api.UElementHandler
+import com.android.tools.lint.detector.api.Category
+import com.android.tools.lint.detector.api.Detector
+import com.android.tools.lint.detector.api.Implementation
+import com.android.tools.lint.detector.api.Issue
+import com.android.tools.lint.detector.api.JavaContext
+import com.android.tools.lint.detector.api.Scope
+import com.android.tools.lint.detector.api.Severity
+import org.jetbrains.uast.UCallExpression
+import org.jetbrains.uast.getContainingUClass
+
+internal val AVOID_USING_ACTIVITY = Issue.create(
+  id = AvoidUsingActivity::class.simpleName!!,
+  briefDescription = "Use the context provided in the lifecycle objects instead.",
+  explanation = "Activity is available here only for use to set the title through the navigable compat.",
```

Comment:
Not: probably don't need to mention Navigable compat here

Reply:
d